client/src/pages/Choice.js

```diff
@@ -25,13 +25,13 @@ const Home = () => {
         </Card>
       </Link>
       <Link to="tour">
-        <Card>
+        <Card hidden>
```

Comment:
If you don't want to render these components at all, you could also use conditional rendering

Reply:
They are not supposed to be shown right now. so no other render method needed ✌️